src/main/resources/application-local.yml

```diff
@@ -33,7 +33,6 @@ faf-client:
     base-url: http://localhost:4444
     scopes: openid offline public_profile upload_map upload_mod lobby
     client-id: faf-java-client
-    timeout-milliseconds: 1500000
```

Comment:
why remove this?

Reply:
no idea

Reply:
It is a deprecated property that is read nowhere

Reply:
Hmm weird thought it was used but you are correct